src/main/java/com/parasoft/demoapp/config/ActiveMQConfig.java

```diff
@@ -1,77 +0,0 @@
-package com.parasoft.demoapp.config;
-
-import java.net.URI;
-
-import lombok.Getter;
-import org.apache.activemq.broker.BrokerService;
-import org.apache.activemq.broker.TransportConnector;
-import org.apache.activemq.command.ActiveMQDestination;
-import org.apache.activemq.command.ActiveMQTopic;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.jms.annotation.EnableJms;
-
-@Configuration
-@EnableJms
-@Getter
-public class ActiveMQConfig {
-
-    public static final String TOPIC_ORDER_APPROVER = "order.approver";
-    public static final String TOPIC_ORDER_PURCHASER = "order.purchaser";
-    public static final String TOPIC_INDUSTRY_CHANGE = "globalPreferences.industryChange";
-    
-    @Value("${spring.activemq.broker-url}")
-    private String embeddedBrokerUrl;
-    
-    @Value("${spring.activemq.embedded-broker-name}")
-    private String embeddedBrokerName;
-    
-    @Value("${spring.activemq.ws-transport-connector-url}")
-    private String wsUrl;
-    
-    @Value("${spring.activemq.stomp-transport-connector-url}")
-    private String stompUrl;
-
-    @Bean
-    public ActiveMQTopic approverTopic() {
-        return new ActiveMQTopic(TOPIC_ORDER_APPROVER);
-    }
-
-    @Bean
-    public ActiveMQTopic purchaserTopic() {
-        return new ActiveMQTopic(TOPIC_ORDER_PURCHASER);
-    }
-
-    @Bean
-    public ActiveMQTopic industryChangeTopic() {
-        return new ActiveMQTopic(TOPIC_INDUSTRY_CHANGE);
-    }
-
-    // Create a embedded ActiveMQ Broker
-    @Bean(initMethod = "start", destroyMethod = "stop")
-    public BrokerService brokerService() throws Exception{
-    	final BrokerService broker = new BrokerService();
-    	broker.setBrokerName(embeddedBrokerName);
-	    broker.setPersistent(false);
-	    // Add Topic destinations
-	    broker.setDestinations(new ActiveMQDestination[]{approverTopic(), purchaserTopic()});
```

Comment:
I understand that for the new queues we may want to customize the name so we don't add those here.
But is there any reason why we removed original line 58 for those two topics? 
The toastr related workflow is broken now I think. @rwang-parasoft

Reply:
The break of the toastr related workflow is not because the remove of line 58. It is because the new convertor(MappingJackson2MessageConverter) we add. The task [PDA-1184](https://jira.parasoft.com/browse/PDA-1184) will handle the break.

The line `broker.setDestinations(new ActiveMQDestination[]{approverTopic(), purchaserTopic()});` is just for setting the tpoic destinations which should be loaded/created on embeded ActiveMQ(PDA) startup. But it will also work well if we  loaded/created those topics when we need them.

So I think we don't need to de anything else.

Reply:
Thanks for you explanation. Let's just merge this pull request.